iso15118/evcc/states/iso15118_20_states.py

```diff
@@ -871,6 +875,11 @@ async def process_message(
                     ISOV20PayloadTypes.MAINSTREAM,
                 )
             else:
+                
+                # EVerest code start #
```

Comment:
I think we don't need these comments, because one could easily search for `EVEREST_CTX` here

Reply:
:+1: